modules/invoicer_invoice/src/Form/InvoicerInvoiceConfigForm.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\invoicer_invoice\Form\InvoicerInvoiceConfigForm.
+ */
+
+namespace Drupal\invoicer_invoice\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Configure file system settings for this site.
+ */
+class InvoicerInvoiceConfigForm extends ConfigFormBase {
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFormId() {
+        return 'invoicer_invoice_config_form';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getEditableConfigNames() {
+        return ['invoicer_invoice.settings'];
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(array $form, FormStateInterface $form_state) {
+        $config = $this->config('invoicer_invoice.settings');
```

Comment:
Yeah, if put this on contructor the people can make better testing. I will think about it.